packages/components/src/TimelineFlightDetail/components/CardHeader.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import {
+  Icon,
+  CarrierLogo,
+  Text,
+  StyleSheet,
+} from '@kiwicom/universal-components';
+
+const capitalize = (text: string) => text[0].toUpperCase() + text.slice(1);
```

Comment:
What do you think to make from this `util/helper function`?
Probably we will need to this feature in more places.

Reply:
+1